### DISCUSSION TITLE
Sync from Plex to Trakt from now

### DISCUSSION THREAD

**Discussion by douggutaby** [Taxel/PlexTraktSync#1014] [Q&A] [answered]
Hi,

I'm new with this tool and I'm afraid of starting for the first time. Almost everything in my plex is checked in/watched in trakt.tv (I'm using an old Plex plugin, but it is not the best). I don't want PlexTraktSync to set them watched again, I just want to use it to sync from now. Can I do that with the config file?

Other topic: Other users use my Plex server and watch my movies. Will PlexTraktSync set them watched in Trakt or just what my user watched? I cannot see user_filter for sync, only for watch command.

Thanks

**Comment by simonc56** [comment, marked as answer]
> I don't want PlexTraktSync to set them watched again, I just want to use it to sync from now. 

PlexTraktSync `sync `command will not set them watched again if it's already watched in trakt. No worry.

> Will PlexTraktSync set them watched in Trakt or just what my user watched?

Just your user.

> I cannot see user_filter for sync

`sync` is always for one user.
`watch` has user_filter option to choose one or all users.

**Comment by douggutaby** [comment]
Thanks, I will set this up and start sync.
